frontends/mit-open/src/pages/ListDetailsPage/ListDetailsPage.tsx

```diff
@@ -0,0 +1,111 @@
+import React from "react"
+import { Container, Grid, Button, BannerPage } from "ol-components"
+import EditIcon from "@mui/icons-material/Edit"
+import SwapVertIcon from "@mui/icons-material/SwapVert"
+
+import { useToggle, pluralize, MetaTags } from "ol-utilities"
+import { GridColumn, GridContainer } from "@/components/GridLayout/GridLayout"
+
+import ItemsListing from "./ItemsListing"
+import type { LearningResourceListItem } from "./ItemsListing"
+
+type OnEdit = () => void
+type ListDetailsPageProps = {
+  listType: string
+  title: string | undefined
+  description: string | null | undefined
```

Comment:
```suggestion
  title?: string
  description?: string | null
```

IMO this is easier to read. There is also a slight difference:
```ts
type A = { x: number, y: number | undefined }
type B = { x: number, y?: number }  

const a1: A = { x: 1 } // ERROR
const a2: A =  { x: 1, y: undefined } // OK

const b1: B = { x: 1 } // OK
const b2: B =  { x: 1, y: undefined } // OK
```